src/schemas/Array.js

```diff
@@ -21,10 +21,10 @@ export const normalize = (schema, input, parent, key, visit, addEntity) => {
   return values.map((value, index) => visit(value, parent, key, schema, addEntity));
 };
 
-export const denormalize = (schema, input, unvisit, getDenormalizedEntity) => {
+export const denormalize = (schema, input, unvisit) => {
   schema = validateSchema(schema);
-  return Array.isArray(input) ?
-    input.map((entityOrId) => unvisit(entityOrId, schema, getDenormalizedEntity)) :
+  return (input && input.map) ?
```

Comment:
`Array.isArray` returns false for immutable lists. This change makes it work for both mutable `Array`s and immutable `List`s and doesn't break any existing tests.